bgpd/bgpd.c

```diff
@@ -2951,6 +2951,8 @@ static struct bgp *bgp_create(as_t *as, const char *name,
 #endif /* ENABLE_BGP_VNC */
 
 	for (afi = AFI_IP; afi < AFI_MAX; afi++) {
+		bgp->vpn_policy[afi].bgp = bgp;
```

Comment:
Are these a fix for something else?  They seem odd for what this commit is doing from what I have looked at so far

Reply:
Part of client adaptation to use label pool. Will be split into separate commit per our discussion.